.changeset/config.json

```diff
@@ -9,9 +9,11 @@
   "commit": false,
   "fixed": [
     [
+      "@commercetools-applications/*",
       "@commercetools-backend/*",
       "@commercetools-frontend/*",
-      "!@commercetools-frontend/codemod"
+      "@commercetools-local/*",
+      "@commercetools-website/*"
```

Comment:
Sorry but I don't the inclusion of this packages here. Will this make them visible in the release changesets?

Reply:
They all already have changesets, as they are part of the monorepo workspaces. E.g. https://github.com/commercetools/merchant-center-application-kit/blob/main/website/CHANGELOG.md

This change is just to align the bumped versions as I mentioned here: https://github.com/commercetools/merchant-center-application-kit/pull/2754#discussion_r942769379
Nothing else changes from the current status quo.